engine/src/main/java/org/hibernate/search/query/grouping/SimpleGroup.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.query.grouping;
+
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopDocs;
+import org.hibernate.search.query.engine.spi.DocumentExtractor;
+import org.hibernate.search.query.engine.spi.EntityInfo;
+
+/**
+ * @author Sascha Grebe
+ */
+public class SimpleGroup implements Group {
```

Comment:
I think we usually call this kind of implementations with the suffix impl, like 'GroupImpl'.
Shouldn't this also be in the impl package?

Some javadoc would be nice